cypress/e2e/signIn.cy.js

```diff
@@ -28,7 +28,31 @@ describe('Sign In page', () => {
     homePage.assertHeaderContainUsername(user.username);
   });
 
-  it('should not provide an ability to log in with wrong credentials', () => {
+  it('should not provide an ability to log in with wrong email', () => {
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
+
+    const userEmail = '77' + user.email;
+    signInPage.typeEmail(userEmail);
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
+
+    cy.contains('Login failed!');
+    cy.contains('Invalid user credentials');
+    cy.contains('OK').click();
+  });
+
+  it('should not provide an ability to log in with wrong password', () => {
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
```

Comment:
Similar to the above, you are registering a new user inside a test case that is meant to verify the scenario of logging in with a wrong password. This should be done in a setup block.

Reply:
done